From 1c926873b76882992ca4299a221cda1db9155d76 Mon Sep 17 00:00:00 2001
From: Pardha Kesapragada <pkesapragada@habana.ai>
Date: Thu, 18 Mar 2021 12:35:48 +0200
Subject: [PATCH] [SW-0] gaudi2_arc_gnu: increase the max size of preboot

The below commit has increased the allotted size
of preboot for all platforms except gaudi2.
Hence make the similar change in this platform
as well, to avoid compile error.

[SW-34382] preboot: gaudi: increase sram

Change-Id: Ide93f73d75aa4fa98e2c925b218b22d7a813524f
Signed-off-by: Pardha Kesapragada <pkesapragada@habana.ai>
---

diff --git a/gaudi2_arc_gnu/common.h b/gaudi2_arc_gnu/common.h
index 4596652..33cd632 100644
--- a/gaudi2_arc_gnu/common.h
+++ b/gaudi2_arc_gnu/common.h
@@ -12,7 +12,7 @@
 #define SRAM_SIZE			0x3000000	/* 48MB */
 
 #define PPBOOT_SIZE			0x4000		/* 16KB */
-#define PREBOOT_SIZE			0xC000		/* 48KB */
+#define PREBOOT_SIZE			0xE000		/* 56KB */
 
 #define HL_FW_BOOT_PTRS_BASE		0x80000
 #define HL_SPI_LEGACY_PREBOOT_OFF	0xC0000
